tutorials/Tutorial7_RAG_Generator.ipynb

```diff
@@ -0,0 +1,165 @@
+    "\n",
+    "Paper: https://arxiv.org/abs/2005.11401  \n",
+    "Original Code: https://github.com/huggingface/transformers\n",
+    "\n",
+    "\n",
+    "*Use this* [link](https://colab.research.google.com/github/deepset-ai/haystack/blob/master/tutorials/Tutorial7_RAG_Generator.ipynb) *to open the notebook in Google Colab.*\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Install the latest release of Haystack in your own environment \n",
+    "#! pip install farm-haystack\n",
+    "\n",
+    "# Install the latest master of Haystack\n",
+    "!pip install git+https://github.com/deepset-ai/haystack/\n",
+    "!pip install urllib3==1.25.4"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from haystack import Document\n",
+    "from haystack.document_store.faiss import FAISSDocumentStore\n",
+    "from haystack.generator.transformers import RAGenerator\n",
+    "from haystack.retriever.dense import DensePassageRetriever"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Add documents from which you want generate answers\n",
+    "documents = [\n",
+    "    Document(\n",
+    "        text=\"\"\"Berlin is Germany capital\"\"\"\n",
+    "    ),\n",
+    "    Document(\n",
+    "        text=\"\"\"Berlin is the capital and largest city of Germany by both area and population.\"\"\",\n",
+    "    )\n",
+    "]"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Initialize FAISS document store to documents and corresponding index for embeddings \n",
+    "document_store = FAISSDocumentStore()\n",
+    "\n",
+    "# Initialize DPR Retriever to encode documents, encode question and query documents\n",
+    "retriever = DensePassageRetriever(\n",
+    "    document_store=document_store,\n",
+    "    query_embedding_model=\"facebook/dpr-question_encoder-single-nq-base\",\n",
+    "    passage_embedding_model=\"facebook/dpr-ctx_encoder-single-nq-base\",\n",
+    "    use_gpu=False,\n",
+    "    embed_title=True,\n",
+    "    remove_sep_tok_from_untitled_passages=True\n",
+    ")\n",
+    "\n",
+    "# Initialize RAG Generator\n",
+    "generator = RAGenerator(\n",
```

Comment:
show some additional args here that users might want to configure (with default values)